lib/elixir_internal_certification/keyword/keywords.ex

```diff
@@ -7,7 +7,14 @@ defmodule ElixirInternalCertification.Keyword.Keywords do
 
   alias ElixirInternalCertification.Account.Schemas.User
   alias ElixirInternalCertification.Keyword.Queries.KeywordQuery
+  alias ElixirInternalCertification.Keyword.Schemas.Keyword
   alias ElixirInternalCertification.Repo
+  alias NimbleCSV.RFC4180, as: CSV
+
+  @max_number_of_keywords_per_csv_file Application.compile_env!(
```

Comment:
```suggestion
  @max_keywords_per_upload Application.compile_env!(
```
You don't have to change but will let you decide 😛  It also doesn't tie it to a `csv` file which may change in future (probably not)